roles/sft-server/templates/sftd.unit.j2

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Running service: SFT server
+
+After=nginx.service
+Wants=nginx.service
+
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+
+# maximum size of core files (bytes)
+LimitCORE=infinity
+LimitCORESoft=0
+
+User={{ SFTD_USER }}
+Group={{ SFTD_GROUP }}
+
+ExecStart={{ SFT_BIN_PATH }} \
+            -I '{{ sft_local_ip }}' \
```

Comment:
I'd just hardcode 127.0.0.1 here - we never want to expose the local part of the sftd without the TLS terminating proxy in front - and if we do, the code around would probably look very different, too.

Reply:
Since it's a `sftd` option, to me it feels right to set this through a variable. But I suppose it could make sense to make it more static by declaring this as a *constant*. Adjusted.